Traceback (most recent call last):
  File "/project2/tas1/pragallva/Fall_quarter_2018//codes/python_scripts/am2/convert_to_pressure_coord.py", line 67, in <module>
    p_sfc=v_var['ps'][:].transpose(1,0,2)
NameError: name 'v_var' is not defined
Traceback (most recent call last):
  File "/project2/tas1/pragallva/Fall_quarter_2018//codes/python_scripts/am2/convert_to_pressure_coord.py", line 67, in <module>
    p_sfc=v_var['ps'][:].transpose(1,0,2)
NameError: name 'v_var' is not defined
Traceback (most recent call last):
  File "/project2/tas1/pragallva/Fall_quarter_2018//codes/python_scripts/am2/convert_to_pressure_coord.py", line 67, in <module>
    p_sfc=v_var['ps'][:].transpose(1,0,2)
NameError: name 'v_var' is not defined
Traceback (most recent call last):
  File "/project2/tas1/pragallva/Fall_quarter_2018//codes/python_scripts/am2/convert_to_pressure_coord.py", line 67, in <module>
    p_sfc=v_var['ps'][:].transpose(1,0,2)
NameError: name 'v_var' is not defined
Traceback (most recent call last):
  File "/project2/tas1/pragallva/Fall_quarter_2018//codes/python_scripts/am2/convert_to_pressure_coord.py", line 67, in <module>
    p_sfc=v_var['ps'][:].transpose(1,0,2)
NameError: name 'v_var' is not defined
Traceback (most recent call last):
  File "/project2/tas1/pragallva/Fall_quarter_2018//codes/python_scripts/am2/convert_to_pressure_coord.py", line 67, in <module>
    p_sfc=v_var['ps'][:].transpose(1,0,2)
NameError: name 'v_var' is not defined
Traceback (most recent call last):
  File "/project2/tas1/pragallva/Fall_quarter_2018//codes/python_scripts/am2/convert_to_pressure_coord.py", line 67, in <module>
    p_sfc=v_var['ps'][:].transpose(1,0,2)
NameError: name 'v_var' is not defined
Traceback (most recent call last):
  File "/project2/tas1/pragallva/Fall_quarter_2018//codes/python_scripts/am2/convert_to_pressure_coord.py", line 67, in <module>
    p_sfc=v_var['ps'][:].transpose(1,0,2)
NameError: name 'v_var' is not defined
Traceback (most recent call last):
  File "/project2/tas1/pragallva/Fall_quarter_2018//codes/python_scripts/am2/convert_to_pressure_coord.py", line 67, in <module>
    p_sfc=v_var['ps'][:].transpose(1,0,2)
NameError: name 'v_var' is not defined
Traceback (most recent call last):
  File "/project2/tas1/pragallva/Fall_quarter_2018//codes/python_scripts/am2/convert_to_pressure_coord.py", line 67, in <module>
    p_sfc=v_var['ps'][:].transpose(1,0,2)
NameError: name 'v_var' is not defined
Traceback (most recent call last):
  File "/project2/tas1/pragallva/Fall_quarter_2018//codes/python_scripts/am2/Calculate_baroclinicity.py", line 76, in <module>
    Z_T_Q_data.append(load(source+"ht_temp_sphum.hkl"))   
  File "/project2/tas1/pragallva/Fall_quarter_2018//codes/python_scripts/am2/Calculate_baroclinicity.py", line 38, in load
    dictionary = hkl.load(filename)
  File "/home/pragallva/Project1/lib/python2.7/site-packages/hickle/hickle.py", line 491, in load
    with file_opener(fileobj) as h5f:
  File "/home/pragallva/Project1/lib/python2.7/site-packages/hickle/hickle.py", line 173, in file_opener
    h5f = h5.File(filename, mode)
  File "/home/pragallva/Project1/lib/python2.7/site-packages/h5py/_hl/files.py", line 312, in __init__
    fid = make_fid(name, mode, userblock_size, fapl, swmr=swmr)
  File "/home/pragallva/Project1/lib/python2.7/site-packages/h5py/_hl/files.py", line 142, in make_fid
    fid = h5f.open(name, flags, fapl=fapl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper (/tmp/pip-install-6ALcG4/h5py/h5py/_objects.c:2846)
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper (/tmp/pip-install-6ALcG4/h5py/h5py/_objects.c:2804)
  File "h5py/h5f.pyx", line 78, in h5py.h5f.open (/tmp/pip-install-6ALcG4/h5py/h5py/h5f.c:2120)
IOError: Unable to open file (unable to open file: name = '/project2/tas1/pragallva/Fall_quarter_2018/post_process_data/am2/data_in_pres_coord/HC0_la16m_oc16m0/ht_temp_sphum.hkl', errno = 2, error message = 'No such file or directory', flags = 0, o_flags = 0)
